menu.sh

```diff
@@ -56,7 +56,7 @@ function printMenu {
     echo "9) Build Siri2GTFS-RT"
     echo "10) Build Digitransit-ui"
     echo "12) Build Postgre database"
-    echo "13) Build Vector Map Server"
+    echo "13) "
```

Comment:
Yes, we no longer have a separate vector map server, do you have a better title in mind?